parser/ast/misc.go

```diff
@@ -2479,6 +2483,15 @@ func (n *AdminStmt) Restore(ctx *format.RestoreCtx) error {
 		} else if n.StatementScope == StatementScopeGlobal {
 			ctx.WriteKeyWord("FLUSH GLOBAL PLAN_CACHE")
 		}
+	case AdminPause:
+		ctx.WriteKeyWord("PAUSE ")
+		ctx.WritePlain(strings.ToUpper(n.TaskType))
```

Comment:
```suggestion
		ctx.WriteName(n.TaskType)
```

If the task type is designed to be a "name", we can use the restored flag to control whether to output in upper case.